### DISCUSSION TITLE
Trying to add PublicFolder, but it breaks my API routes...

### DISCUSSION THREAD

**Discussion by Lroy** [sukona/Grapevine#223] [Q&A] [answered]
I am positive that I'm doing something wrong here, or in the wrong order, but I cannot seem to figure it out.

I add some API routes using Server.Router.Register(route).  That works fine.

If I don't add any public folders, the API routes work.  If I add a public folder, the API routes no longer work, and the logger suggests that Grapevine is looking for a file with the name of my API route.

What am I doing wrong here?

<img width="818" alt="Screen Shot 2021-02-05 at 11 00 32 AM" src="https://user-images.githubusercontent.com/290178/107058075-251db980-67a2-11eb-8885-4ddedf787875.png">

**Comment by Lroy** [comment]
The logger returns this, if I uncomment 'server.PublicFolder = new PublicFolder("public","/")'.

<img width="792" alt="Screen Shot 2021-02-05 at 11 08 00 AM" src="https://user-images.githubusercontent.com/290178/107058418-978e9980-67a2-11eb-9077-910546bd9896.png">
<img width="619" alt="Screen Shot 2021-02-05 at 11 09 07 AM" src="https://user-images.githubusercontent.com/290178/107058430-9a898a00-67a2-11eb-9837-94b8d5b7d5d3.png">

**Comment by Lroy** [comment, marked as answer]
Ok,

So it looks like I am specifying "/" as the root of the "public" folder, and it should actually be "".

So I added another public folder at "C:\temp\Sample", and that route prefix should simply be "Sample", not "/Sample".

I think I finally have is straight!

Everything else seems to be working great!

**Reply by scottoffen** [reply]
You can leave out the `prefix` parameter in the constructor entirely.
